advocacy_docs/repos/reference/repositories_by_name.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Repositories by name
+navTitle: Repositories by name
+description: A list of repositories and the subscriptions required to access them
+---
+
+## Current major repositories
+
+For current supported products from EDB, the following repositories are available.
+
+| Repository             | Required Subscriptions | Contents                |
+|------------------------|------------------------|-------------------------|
+| community360           | Community360           | Supported Open Source   |
+| standard               | Standard               | Standard subscription   |
+| enterprise             | Enterprise             | Enterprise subscription |
+| postgres_extended      | PGD                    | 2ndQPostgres            |
```

Comment:
Can we include a note along the lines of 'These are the legacy Postgres Extended packages previously known as 2ndQPostgres. If you are looking fr the current Postgres Extended packages, these are available in enterprise and standard repos.

Reply:
Moved it down into legacy.